domain/camera/src/main/java/com/google/jetpackcamera/domain/camera/CameraXCameraUseCase.kt

```diff
@@ -834,6 +883,34 @@ constructor(
             getResolutionSelector(sessionSettings.aspectRatio)
         )
 
+        if (currentSettings.value?.lowLightBoost == LowLightBoost.ENABLED) {
+            Log.d(TAG, "Adding Control AE_MODE_ON_LOW_LIGHT_BOOST_BRIGHTNESS_PRIORITY")
+            val camera2Interop = Camera2Interop.Extender(previewUseCaseBuilder)
+            if (Build.VERSION.SDK_INT >= 35) {
+                camera2Interop.setCaptureRequestOption(
+                    CaptureRequest.CONTROL_AE_MODE,
+                    CameraMetadata.CONTROL_AE_MODE_ON_LOW_LIGHT_BOOST_BRIGHTNESS_PRIORITY
+                )
+                camera2Interop.setSessionCaptureCallback(object : CaptureCallback() {
+                    override fun onCaptureCompleted(
+                        session: CameraCaptureSession,
+                        request: CaptureRequest,
+                        result: TotalCaptureResult
+                    ) {
+                        super.onCaptureCompleted(session, request, result)
+                        val boostState = result.get(CaptureResult.CONTROL_LOW_LIGHT_BOOST_STATE)
+                        if (boostState == CameraMetadata.CONTROL_LOW_LIGHT_BOOST_STATE_ACTIVE) {
+                            Log.d(TAG, "LLB Active")
+                            _lowLightBoostActiveStatus.value = true
+                        } else {
+                            Log.d(TAG, "LLB Inactive")
+                            _lowLightBoostActiveStatus.value = false
+                        }
+                    }
+                })
```

Comment:
We should add this callback regardless of whether LLB is enabled. Technically the `CONTROL_AE_MODE` can be set at any time (without needing to rebind use cases). So it could go from the `DISABLED` -> `ENABLED` state without needing to recreate / rebind `Preview`.